src/sentry/event_manager.py

```diff
@@ -399,6 +399,8 @@ def save(self, project_id, raw=False, assume_normalized=False, start_time=None,
             "last_seen": job["event"].datetime,
             "first_seen": job["event"].datetime,
             "active_at": job["event"].datetime,
+            "metadata": dict(job["event_metadata"]),
```

Comment:
Would it be valid to use `job["data"]["metadata"]` here instead, and never set `job["event_metadata"]`? I'm asking because I'm confused by the number of fields containing the same or similar data.

Reply:
yeah we can do that

Reply:
Actually we can't. `job['data']['metadata']` contains extra data that we do not want in the group